doc/reference/reference_sql/sql-features.rst

```diff
@@ -14,443 +13,1032 @@ else we will mark it "Fail".
+        *   -   E141-04   
+            -   Basic FOREIGN KEY constraint with the NO ACTION default
+                for both referential delete and referential update actions
+            -   ``CREATE TABLE t11 (s0 INT PRIMARY KEY, s1 INT REFERENCES t10);``
+            -   :ref:`Okay <sql_foreign_key>`.   
+        *   -   E141-06   
+            -   CHECK constraints
+            -   ``CREATE TABLE t12 (s1 INT PRIMARY KEY, s2 INT, CHECK (s1 = s2));``
+            -   Okay.  
+        *   -   E141-07   
+            -   Column defaults
+            -   ``CREATE TABLE t13 (s1 INT PRIMARY KEY, s2 INT DEFAULT -1);``   
+            -   Okay.
+        *   -   E141-08   
+            -   NOT NULL inferred on primary key
+            -   ``CREATE TABLE t14 (s1 INT PRIMARY KEY);``   
+            -   Okay. We are unable to insert NULL although we don't
+                explicitly say the column is NOT NULL.   
+        *   -   E141-10   
+            -   Names in a foreign key can be specified in any order
+            -   ``CREATE TABLE t15 (s1 INT, s2 INT, PRIMARY KEY (s1,s2));``
+                ``CREATE TABLE t16 (s1 INT PRIMARY KEY, s2 INT, FOREIGN KEY (s2,s1) REFERENCES t15 (s1,s2));``   
+            -   Okay.  
+   
+   
+E151, Transaction support   
+-------------------------
+
+..  container:: table
 
+    ..  list-table::
+        :widths: 10 30 30 30
+        :header-rows: 1
 
+        *   -   Feature ID
+            -   Feature
+            -   Example
+            -   Tests
+        *   -   E151-01   
+            -   COMMIT statement
+            -   ``COMMIT;``   
+            -   Fail. Tarantool supports   
+                :ref:`COMMIT <sql_commit>` but it is necessary to say 
+                :ref:`START TRANSACTION <sql_start_transaction>` first.
+        *   -   E151-02   
+            -   ROLLBACK statement
+            -   ``ROLLBACK;``
+            -   :ref:`Okay <sql_rollback>`.   
+   
+   
+E152, Basic SET TRANSACTION statement
+-------------------------------------
 
+..  container:: table
+
+    ..  list-table::
+        :widths: 10 30 30 30
+        :header-rows: 1
+
+        *   -   Feature ID
+            -   Feature
+            -   Example
+            -   Tests
+        *   -   E152-01   
+            -   SET TRANSACTION statement: ISOLATION SERIALIZABLE clause
+            -   ``SET TRANSACTION ISOLATION LEVEL SERIALIZABLE;``   
+            -   Fail. Syntax error.   
+        *   -   E152-02
+            -   SET TRANSACTION statement: READ ONLY and READ WRITE clauses
+            -   ``SET TRANSACTION READ ONLY;``   
+            -   Fail. Syntax error.   
+         
+
+E*, Other
+---------
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 10 30 30 30
+        :header-rows: 1
+
+        *   -   Feature ID
+            -   Feature
+            -   Example
+            -   Tests
+        *   -   E153   
+            -   Updatable queries with subqueries   
+            -
+            -
```

Comment:
Hello @pgulutzan,
There's no example and no test results for E153, is that correct? This table contains tests results for all other items, even cursors, though Tarantool doesn't support them and we have no examples for them.